airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json

```diff
@@ -32,8 +32,8 @@
         "order": 2
       },
       "schema": {
-        "title": "Schema",
-        "description": "Unless specifically configured, the usual value for this field is \"public\".",
+        "title": "Default Schema",
```

Comment:
Result:
<img width="806" alt="Screen Shot 2021-04-04 at 6 10 56 PM" src="https://user-images.githubusercontent.com/9772859/113505391-1de01580-9571-11eb-904c-0ad744157e13.png">

Reply:
>  source does not specify a schema

i would suggest "...source does not specify a namespace" instead of "schema". we are trying to add namespace as a primitive into our system, so we shouldn't be afraid to use it. saying schema sorta assumes that the source is a database.

Reply:
good point. making the change.